dist/build.sh

```diff
@@ -20,7 +20,7 @@ arch=$(cat pkg/$dist-$ver.arch)
 
 gpg --sign --armor --detach-sign pkg/$pkg
 
-aws s3 cp pkg/$pkg s3://dist.metrist.io/orchestrator/$dist/
-aws s3 cp pkg/$pkg.asc s3://dist.metrist.io/orchestrator/$dist/
-echo $pkg | aws s3 cp - s3://dist.metrist.io/orchestrator/$dist/$ver.$arch.latest.txt
-aws cloudfront create-invalidation --distribution-id E1FRDOED06X2I8 --paths "/orchestrator/$dist/*"
+# aws s3 cp pkg/$pkg s3://dist.metrist.io/orchestrator/$dist/
+# aws s3 cp pkg/$pkg.asc s3://dist.metrist.io/orchestrator/$dist/
+# echo $pkg | aws s3 cp - s3://dist.metrist.io/orchestrator/$dist/$ver.$arch.latest.txt
+# aws cloudfront create-invalidation --distribution-id E1FRDOED06X2I8 --paths "/orchestrator/$dist/*"
```

Comment:
```suggestion
aws s3 cp pkg/$pkg s3://dist.metrist.io/orchestrator/$dist/
aws s3 cp pkg/$pkg.asc s3://dist.metrist.io/orchestrator/$dist/
echo $pkg | aws s3 cp - s3://dist.metrist.io/orchestrator/$dist/$ver.$arch.latest.txt
aws cloudfront create-invalidation --distribution-id E1FRDOED06X2I8 --paths "/orchestrator/$dist/*"
```
Uncomment when review is done